code/controllers/subsystem/ticker.dm

```diff
@@ -363,6 +364,16 @@ SUBSYSTEM_DEF(ticker)
 				to_chat(N, "Captainship not forced on anyone.")
 			CHECK_TICK
 
+/datum/controller/subsystem/ticker/proc/apply_round_modifiers()
+	for(var/mob/dead/new_player/N in GLOB.player_list)
+		var/mob/living/L = N.new_character
+		if(!istype(L))
+			continue
+
+		for(var/d in SSround_modifiers.active_modifiers)
```

Comment:
Check if there are any active modifiers first, stops from looping through all players only to do nothing.